client/src/app/site/site.component.ts

```diff
@@ -106,56 +75,7 @@ export class SiteComponent extends BaseComponent implements OnInit {
     public ngOnInit(): void {
         this.subscriptions.push(...this.getRouterSubscriptions());
         // observe the mainMenuService to receive toggle-requests
-        this.mainMenuService.toggleMenuSubject.subscribe((value: void) => this.toggleSideNav());
-
-        // get a translation via code: use the translation service
-        // this.translate.get('Motions').subscribe((res: string) => {
-        //      console.log('translation of motions in the target language: ' + res);
-        //  });
-
-        // TODO: Remove this, when the ESR version of Firefox >= 64.
-        const agent = navigator.userAgent.toLowerCase();
-        if (agent.indexOf('firefox') > -1) {
-            const index = agent.indexOf('firefox') + 8;
-            const version = +agent.slice(index, index + 2);
-
-            if (version < 64) {
-                const sideNav = document.querySelector(
-                    'mat-sidenav.side-panel > div.mat-drawer-inner-container'
-                ) as HTMLElement;
-                sideNav.style.overflow = 'hidden';
-                sideNav.addEventListener('MozMousePixelScroll', (event: any) => {
-                    sideNav.scrollBy(0, event.detail);
-                });
-            }
-        }
```

Comment:
I have noticed, that the Firefox ESR-version is 78, now. So, I deleted this code.